plugins/source/gitlab/Makefile

```diff
@@ -0,0 +1,39 @@
+# Test unit
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+# Install tools
+.PHONY: install-tools
+install-tools:
+	@echo Installing tools from tools/tool.go
+	@cat tools/tool.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+
+# Install pre-commit hooks. This requires pre-commit to be installed (https://pre-commit.com/)
+.PHONY: install-hooks
+install-hooks:
+	pre-commit install
+
+.PHONY: lint
+lint:
+	golangci-lint run --config ../../.golangci.yml
+
+.PHONY: gen-code
+gen-code:
+	grep -rl '// Code generated by codegen; DO NOT EDIT.' resources/services/* | xargs rm
+	go run codegen/main.go
+
+.PHONY: gen-mocks
+gen-mocks:
+	go install github.com/golang/mock/mockgen@v1.6.0
```

Comment:
`install-tools` already installs mockgen (if you include the `tools/too.go`) so this shouldn't run go install?

Reply:
This is taken out of the AWS makefile... Trying to not reinvent the wheel/ keep consistent with other plugins...

Reply:
That's why I suggested the thing in the first place. AWS makefile also has that mistake apparently, and it's the only plugin that has that other than this PR.

Reply:
Azure makefile also has something similar though... maybe we should remove tools/tool.go target? not sure

Reply:
I have taken your advice: removed the explicit install and using install tools target